data/147/772/729/3/1477727293.geojson

```diff
@@ -0,0 +1,63 @@
+{
+  "id": 1477727293,
+  "type": "Feature",
+  "properties": {
+    "edtf:cessation":"uuuu",
+    "edtf:inception":"uuuu",
+    "geom:area":0.00888,
+    "geom:area_square_m":73168893.805513,
+    "geom:bbox":"-122.770280494,48.1542450987,-122.623963357,48.2591328032",
+    "geom:latitude":48.214014,
+    "geom:longitude":-122.687716,
+    "iso:country":"US",
+    "lbl:latitude":48.211316,
+    "lbl:longitude":-122.674423,
+    "mps:latitude":48.211316,
+    "mps:longitude":-122.674423,
+    "mz:hierarchy_label":1,
+    "mz:is_current":1,
+    "mz:is_landuse_aoi":1,
+    "name:ceb_x_preferred":[
```

Comment:
We should consider blocking these `ceb` names from Wikidata, they continue to have a bad bot over there.

Reply:
I'm going to track this independent of this PR, as both the `ceb` and `vol` language tags are suspect.
I'll likely merge as-is, then iterate over all admin records to remove/rework those name properties.